gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -144,7 +145,7 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
 
     // off-heap bytes
     if (!conf.contains(GlutenConfig.GLUTEN_OFFHEAP_SIZE_KEY)) {
-      throw new UnsupportedOperationException(s"${GlutenConfig.GLUTEN_OFFHEAP_SIZE_KEY} is not set")
+      throw new GlutenException(s"${GlutenConfig.GLUTEN_OFFHEAP_SIZE_KEY} is not set")
```

Comment:
`GlutenNotSupportException` ?

Reply:
Looks the exception here also cannot make fall back happen, but just let the execution fail. So I think `GlutenException` may be good. Thanks!